html/index.html

```diff
@@ -10,6 +10,40 @@
 
 <body>
     {{template "navbar"}}
+
+    <div class="container">
```

Comment:
I've tried out the latest code because I was curious and it might be good to limit the maximum width of the buttons to something like 400px otherwise they are huge on a wide screen desktop monitor.

Reply:
Yes, this is one of the things I was going to fix.

The caret in the navbar is also not right.  It needs to be a separate button to the left of the device name right? One that acts like a proper button, not just part of the device name text.

And it also needs to hide on the index page I imagine?

Reply:
If you're in a hurry for this, I can try and do it today?

Reply:
No major rush. Just when you have time.

The caret in the navbar is ok. It would be slightly nicer if it acted like a button but it doesn't matter too much.